README.md

```diff
@@ -227,6 +228,22 @@ epsagon.init(
     keys_to_ignore=['Request Data', 'Status_Code']
 )
 ```
+
+### Allowed keys
+You can allow data to be sent to epsagon by filtering specific keys in initialization.
+Only keys included in this list will be sent to epsagon.
+Good to know - keys_to_ignore is stronger then keys_to_allow if you use both.
```

Comment:
NOTE - Keys found in keys_to_ignore override this setting

Reply:
fixed